src/kibana/plugins/visualize/editor/vis_options.html

```diff
@@ -1,12 +1,12 @@
 <li class="sidebar-item" ng-show="vis.type.params.editor">
-  <div class="sidebar-item-title" ng-click="showVisOptions = !showVisOptions">
+  <div ng-if="!alwaysShowOptions" class="sidebar-item-title" ng-click="$parent.showVisOptions = !showVisOptions">
```

Comment:
If you switch these ng-if expressions to ng-show then you won't need to reference `$parent`. Otherwise, I would recommend storing the showVisOptions in a way that isn't susceptible to the issue you're trying to work around here.